### ISSUE TITLE
[Feature Request - Web] Is it possible to allow a user to group their related packages so that there is only 1 update for a framework instead of 40? 

### ISSUE THREAD

**Issue by MostHated** [openupm/openupm#2297]
## Feature request
Is it possible to allow a user to group all of their related packages for their framework under a single display/update notification? It is honestly getting to the point where it is annoying/frustrating to have to hit "read" on all the notification that come through RSS, but also discouraging to visit the OpenUPM site to check out packages because there are basically always several pages worth that are all a single framework.

It is nothing against this particular developer (there are a few that have a decent number of related packages, but nowhere near as many), it is good for them that they have such a feature-rich and modular framework, but it is really getting out of hand, as you can see in the images below.

#### Feature category

- [x] Website: improve the way to browse or discover packages.
- [ ] Pipelines: improve the way to publish or manage a package.
- [ ] Registry: improve the registry server.
- [ ] Unity Editor: improve how OpenUPM works with the Unity Editor.
- [ ] Documentation
- [ ] Others

#### How should this be implemented in your opinion?
Perhaps allow the user to tag/label their repo with a specific key/value pair (ex. framework:myframework) that designates they are all part of the same framework, then on the website, just show one entry for the framework itself, that when clicked on, takes you to a page displaying all the packages of that framework, or just shows all packages from that particular developer.
 
#### What problem does this feature solve?
The extreme amount of updates from a single framework.

![](https://i.imgur.com/qR7Rf6d.png)

![](https://i.imgur.com/jn2QIub.gif)

**Comment by favoyang** [comment]
@MostHated thanks for the suggestions. I agree that it can be an option to group certain packages that belong to one "framework" into one aggregated entry in a feed list.

![image](https://user-images.githubusercontent.com/125390/126886849-a02c561c-a7bd-44da-a2dd-fb4f24c3ffa5.png)

**Comment by favoyang** [comment]
But I'm not sure if we could group everything with "Tilia" into one entry. We need to look into the difference between "Tilia Interactions", "Tilia Locomomotors" and others.

**Comment by MostHated** [comment]
Yeah, I didn't quite know how best to suggest trying to go about it as I am not super experienced with web stuff, but thought perhaps repo topic tags could be used if the repo owner was ok with adding them as it basically lets you create your own new URL that groups all your stuff for you.

![](https://i.imgur.com/VpThYtM.png)

I added the topic tag "upmframework-instanceid" to my repo, which then created this:

https://github.com/topics/upmframework-instanceid  

You then get a listing of everything that has that tag. So if I had a bunch of smaller repos and I gave all of them that same tag, they would all be listed under that URL.

The only issue with that, though, is the repo owner would have to add that themselves, just as they have to add their version tag.
